feat(ins): add instance bandwidth overview (#4003)
scaleway/docs-content: compute/instances/reference-content/instances-bandwidth-overview.mdx

tags: instance
dates:
  validation: 2024-11-19
  posted: 2024-11-19
categories:
  - compute
---

This guide provides a complete overview of internet and Block Storage bandwidth for Scaleway Instances.
Understanding these specifications is crucial for optimizing your infrastructure for performance and compatibility.

### Why bandwidth matters

Bandwidth impacts how your applications perform and interact with other systems.
- **Internet bandwidth**: Affects how fast your Instance can send or receive data over the internet.
- **Block bandwidth**: Determines the read/write speed to Block Storage, essential for storage-heavy workloads like databases and analytics.

<Message type="important">
  Bandwidth specifications listed here are for informational purposes. To validate the exact bandwidth capacity of your Instance type, refer to the `block_bandwidth` field in the [Instances API](https://www.scaleway.com/en/developers/api/instance/#path-instance-types-list-instance-types). Use the endpoint `/instance/v1/zones/{zone}/products/servers` to retrieve Instance type specifications.
</Message>

<Message type="tip">

>   To maximize compatibility with [Block Storage Low Latency 15k](https://www.scaleway.com/en/docs/storage/block/), select an Instance with **at least 3 Gbps** of Block bandwidth.

Author: ```suggestion
  To maximize compatibility with [Block Storage Low Latency 15k](https://www.scaleway.com/en/docs/storage/block/), select an Instance with **at least 3 GiB/s** of Block bandwidth.
```

---
meta:
  title: Scaleway Instances internet and Block Storage bandwidth overview
  description: Find detailed information about the internet and Block Storage bandwidth for each Instance type at Scaleway.
content:
  h1: Scaleway Instances internet and Block Storage bandwidth overview
  paragraph: Find detailed information about the internet and Block Storage bandwidth for each Instance type at Scaleway.
tags: instance
dates:
  validation: 2024-11-19
  posted: 2024-11-19
categories:
  - compute
---

This guide provides a complete overview of internet and Block Storage bandwidth for Scaleway Instances.

> Explore the list of Scaleway Instance types categorized by families, with detailed internet and block bandwidth specifications.

Author: ```suggestion
```

---
meta:
  title: Scaleway Instances internet and Block Storage bandwidth overview
  description: Find detailed information about the internet and Block Storage bandwidth for each Instance type at Scaleway.
content:
  h1: Scaleway Instances internet and Block Storage bandwidth overview
  paragraph: Find detailed information about the internet and Block Storage bandwidth for each Instance type at Scaleway.
tags: instance
dates:
  validation: 2024-11-19
  posted: 2024-11-19
categories:
  - compute
---

This guide provides a complete overview of internet and Block Storage bandwidth for Scaleway Instances.

> Explore the list of Scaleway Instance types categorized by families, with detailed Internet and block storage bandwidth specifications.

Author: ```suggestion
Explore the list of Scaleway Instance types categorized by families, with detailed internet and block bandwidth specifications.
```
